### ISSUE TITLE
Add or update 21x tracks

### ISSUE THREAD

**Issue by irabinovitch** [socallinuxexpo/scale-drupal#83]
@kcbraunschweig please add a list of tracks for SCALE 21x when you have a moment.
Once we have those @DrupalPhil and I need to do the following:

- [x] Create tracks in the track taxonomy
- [x] hide tracks by id in the ./sites/all/modules/custom/scale_utility/scale_utility.module 
- [ ] run through the various CFP reports (Eg presentation filtered) and make sure they match the list we expect.

**Comment by kcbraunschweig** [comment]
Here is the 21x track list:

Career Day
Cloud Native
Data on Kubernetes (DoK)
Developer
DevOpsDayLA
Embedded
FOSS@Home
General
HAM Radio & Linux
Kernel & Low Level Systems
Kubernetes Community Day
MySQL
Observability
Open Government
Open Source AI and Applied Science
PostgreSQL@SCaLE (PostgreSQL)
Reproducible and Immutable Software
Security
Systems & Infrastructure
The Next Generation
UpSCaLE

**Comment by irabinovitch** [comment]
@DrupalPhil let me know if you need help adding or hiding these.

**Comment by irabinovitch** [comment]
Added new tracks in the Track taxonomy, and hid the unused old ones in #98

**Comment by irabinovitch** [comment]
@kcbraunschweig take a look and let me know if it works as expected. We might still need to fix a few of the track lists in the CFP reports

**Comment by kcbraunschweig** [comment]
<img width="336" alt="image" src="https://github.com/socallinuxexpo/scale-drupal/assets/739983/679fc453-ca76-456c-92e0-18ea9d071917">
On the cfp form page it looks like all the new tracks are there but some old ones need to be removed. Also I'd prefer to update some so that the text matches exactly what's in the cfp text and what we're tracking elsewhere:

Aerospace -> remove
Data on Kubernetes -> Data on Kubernetes (DoK)
FOSS @ HOME -> FOSS@Home
HAM Radio -> HAM Radio & Linux
Next Generation -> The Next Generation
Open Data -> remove
Open Medical -> remove
PostgreSQL -> PostgreSQL@SCaLE (PostgreSQL)
Systems and Infrastructure -> Systems & Infrastructure
Test -> this one always confuses people that think it is for talks about Testing so can we change it to "Test Topic (do not use)" or something?
Workshops -> remove (workshops should be submitted in their topic area)

**Comment by irabinovitch** [comment]
Try refreshing? Here's what I see.  All the ones you flagged for removal should already be gone via #98 
![image](https://github.com/socallinuxexpo/scale-drupal/assets/289654/c0673b86-7b4d-4c5b-b7c9-f2b73dc73286)

**Comment by kcbraunschweig** [comment]
ah another refresh fixed it, yes all the removed ones I see removed now

**Comment by kcbraunschweig** [comment]
Ok 2 internal pages that still need track filter updates:
Topic filter on https://www.socallinuxexpo.org/presentations-filtered
and https://www.socallinuxexpo.org/presentation-votes

- remove Aerospace
- remove Containers and Orchestrators
- add Data on Kubernetes
- remove LibreGraphics
- remove Open Data
- remove Open Medical
- remove Open Source in Enterprises
- add Open Source AI and Applied Science
- add Reproducible and Immutable Software

**Comment by irabinovitch** [comment]
I dont think we should hide these on the internal pages.

**Comment by kcbraunschweig** [comment]
I don't understand? Why have filters that don't work for tracks that don't exist? Or are you saying there's only 1 filter list regardless of year so removing some means we can't use them for previous years? If that's the case, then agreed, don't remove any, just add the new ones.
But for the redesign then I'd ask that the filters be set based on the tracks for that year rather than globally. Otherwise the list will just grow unbounded over time.

**Comment by irabinovitch** [comment]
I dont believe those pages have any sense of a filter list. So if we remove them, we'd be removing the ability to filter on them in any year.

**Comment by irabinovitch** [comment]
Closing since we're now planning for 23x.
